src/Select.js

```diff
@@ -635,6 +672,16 @@ export default class Select extends Component<Props, State> {
   // ==============================
   // Helpers
   // ==============================
+  announceAriaLiveSelection = ({ event, context }: { event: string, context: ValueEventContext }) => {
+    this.setState({
+      ariaLiveSelection: valueEventAriaMessage(event, context),
+    });
+  }
+  announceAriaLiveContext = ({ event, context }: { event: string, context?: InstructionsContext }) => {
+    this.setState({
+      ariaLiveContext: instructionsAriaMessage(event, { ...context, label: this.props['aria-label'] }),
```

Comment:
this calls the instructionsAriaMessage helper with the passed in event and context to resolve to the next value of ariaLiveContext in state.